src/main/puzzle/piece/piece-manager.gd

```diff
@@ -143,9 +138,9 @@ func _spawn_piece() -> void:
 	
 	# apply initial infinite DAS
 	var initial_das_dir := 0
-	if _input_left_frames >= piece_speed.delayed_auto_shift_delay:
+	if _input_left_frames >= PieceSpeeds.current_speed.delayed_auto_shift_delay:
 		initial_das_dir -= 1
-	if _input_right_frames >= piece_speed.delayed_auto_shift_delay:
+	if _input_right_frames >= PieceSpeeds.current_speed.delayed_auto_shift_delay:
```

Comment:
Document this change in the commit comment; piece_speed was moved from piece-manager.gd into piece-speeds.gd

Reply:
Updated commit comment, `Moved piece_speed from piece-manager.gd into piece-speeds.gd`.